Basic/DistributedAuthoritySocialHub/Assets/Editor/Tutorials/Tutorial0_SetupUGS/1-Verify And Setup UGS.asset

```diff
@@ -0,0 +1,122 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: ff771ccdf4150419d9ff4d342b069aae, type: 3}
+  m_Name: 1-Verify and set up Unity Gaming Service (UGS)
+  m_EditorClassIdentifier: 
+  Title:
+    m_Untranslated: Configure Unity Gaming Services (UGS)
+  m_Paragraphs:
+    m_Items:
+    - m_Type: 6
+      Title:
+        m_Untranslated: 
+      Text:
+        m_Untranslated: 
+      m_CodeSample: 
+      m_PostInstructionImage: {fileID: 0}
+      m_Tutorial: {fileID: 0}
+      m_Image: {fileID: 0}
+      m_VideoUrl: 
+      m_Video: {fileID: 0}
+      m_CriteriaCompletion: 0
+      m_Criteria:
+        m_Items: []
+      m_MaskingSettings:
+        m_MaskingEnabled: 1
+        m_UnmaskedViews: []
+      m_Summary: 
+      m_Description: 
+      m_InstructionBoxTitle: 
+      m_InstructionText: 
+      m_TutorialButtonText: 
+    - m_Type: 0
+      Title:
+        m_Untranslated: 
+      Text:
+        m_Untranslated: "The <b>Distributed Authority Social Hub sample</b> uses
+          multiple Unity Cloud services for ease of use and scalability. \n\nTo
+          use this sample, you must first link your <b>Unity Project</b> to <b>Unity
+          Cloud.</b> This tutorial guides you through the steps to <b>link your
+          project to Unity Cloud.</b>\n\n<b>Note:</b> <i>You can also
+         connect a project to Unity Cloud in the <b>Unity Hub</b>. To do this, create a
+          new project and select the <b>Connect to Unity Cloud</b> checkbox.</i>"
```

Comment:
What's the point of this "Note:" paragraph, considering that users are already in this sample? We don't want them to go away and create a new project while they're doing this tutorial, we want them to keep exploring this sample.

Reply:
It's still valuable information, especially for a very new user.
Also, the bulk of this tutorial is shared from the [VR template](https://github.cds.internal.unity3d.com/unity/com.unity.template.vr-multiplayer/tree/release/6000.0), so for things like content and styling, I think I'll leave the same.

Reply:
> It's still valuable information, especially for a very new user.

Ok, and that's what we have online documentation for.

However, a tutorial of a sample should be relevant about the content of the sample. Otherwise, for the same principle, we should link to any "useful" documentation resource out there, no?

What value does that paragraph provide to a user of this sample?

> the bulk of this tutorial is shared from the [VR template](https://github.cds.internal.unity3d.com/unity/com.unity.template.vr-multiplayer/tree/release/6000.0), so for things like content and styling, I think I'll leave the same.

Doesn't sound a good reason to go against [design guidelines](https://docs.google.com/presentation/d/1aEJkrIhvJ4hf-IXOghLvl0Y3_u7-hPd0aHlqa3RKddc/edit#slide=id.p) designed by professionals. That template should instead be adjusted to meet them, not the opposite.

Reply:
That's not what I'm suggesting. It's worth mentioning since it's relevant to the particular step that they have just completed, which could also be accomplished in 2 different ways.

Could you please point out the problematic bolding instances?

Reply:
> It's worth mentioning since it's relevant to the particular step that they have just completed, which could also be accomplished in 2 different ways.

If they go to the hub and create a new project and connect it to the Cloud, can they move forward in the DA Sample's tutorial?

if not, then that action is not relevant to the sample.

> Could you please point out the specific problematic bolding instances?

"Connect to Unity Cloud" is an example. "Connect to" is not a unity-specific term, nor a path to a window in the editor. There's many of those "unnecessary bolding" scattered in the content, and as per the guidelines, they should not be there (same goes for _italic_ text)

Reply:
Got rid of a few bolding instances and got rid of italics. Let me know if there's any problematic instances left.